example/com/example/BUILD.bazel

```diff
@@ -5,3 +5,6 @@ proto_library(name='aa_proto', srcs=['aa.proto'], visibility=['//visibility:publ
 java_proto_library(name='aa_proto_java', deps=[':aa_proto'], visibility=['//visibility:public'])
 py_proto_library(name='aa_proto_py', deps=[':aa_proto'], visibility=['//visibility:public'])
 # ---- END BZL_GEN_BUILD_GENERATED_CODE ---- no_hash
+# ---- BEGIN BZL_GEN_BUILD_GENERATED_CODE ---- no_hash
+py_library(name='hello', srcs=['hello.py'], deps=['@@rules_python~0.24.0~pip~pip_39_pandas//:pkg'], visibility=['//visibility:public'])
+# ---- END BZL_GEN_BUILD_GENERATED_CODE ---- no_hash
```

Comment:
Is it expected to have two different generated sections with the same name?

I had thought we'd only have one of these, or else we'd use two different names for the different parts. (That way each generated section could be safely updated independently without affecting any others.)

Reply:
No the target name must be unique within a package. This package demonstrates that bzl-gen-build can be configured to produce "hello" package even though it's inside of the "example" package. Normally we make a target based on the package (directory) name not the source file name.

Reply:
Sorry i wasn't more clear. I was actually referring to the generated comment section names (i.e. "BZL_GEN_BUILD_GENERATED_CODE"). If we have to generators "sharing" a file and want them to safely be able to update their own section they'd need to be able to distinguish those sections somehow.

It's not really a big deal,  and it would be easy to change I think, I just wanted to point it out.

Reply:
At least in the way it's done right now, the polyglot support is implemented by simply appending output after another, so yea, we end up with multiple codegen regions. I guess it's something we can follow up.